### DISCUSSION TITLE
403 On Intermediate table

### DISCUSSION THREAD

**Discussion by dellean-santos-paypal** [directus/directus#17515] [Archive]
### Describe the Bug

We got 403 request error when click to see a file.

<img width="857" alt="Screenshot 2023-02-14 at 15 58 30" src="https://user-images.githubusercontent.com/108411157/218831912-b4ef04b4-d167-42f7-b36a-afaa7d2068bb.png">

### To Reproduce

- Create a new collection
- Create a new field as a files type.
- Create a new record of created collection
- Add a new file and click to edit
- Got 403 Error 
<img width="641" alt="Screenshot 2023-02-14 at 16 00 39" src="https://user-images.githubusercontent.com/108411157/218832164-91a25a7a-85db-479c-bf38-54b2eac934b8.png">

As I try to understand the error, the directus create an intermediate table "colletion_created_files" and try to fetch asset data by calling API_PATH/items/<intermediate_table>/<directus_files_id> instead <intermediate_table_record_id>

### Errors Shown

```json
{
  "errors": [
    {
      "message": "You don't have permission to access this.",
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ]
}
````

### What version of Directus are you using?

9.22.4

### What version of Node.js are you using?

18.14.0

### What database are you using?

postgres

### What browser are you using?

Chrome 

### How are you deploying Directus?

Self Hosted On Heroku

**Comment by rijkvanzanten** [comment]
Does your current user have permissions to read the items from `colletion_created_files`? Does the record you're trying to open from `colletion_created_files` exist? 🤔
